### PR TITLE
BLD-5962 ensure choco is installed first then install git.install

### PR DESCRIPTION
**Pull request by sjackson-ccdc**

### CODE REVIEW

File: tasks/Windows.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ---
+- name: Ensure Chocolatey is installed
```
Comment (siong-chin) [inline tasks/Windows.yml:2]: I don't think this is needed. According to the doc

> If Chocolatey is missing from the system, the module will install it.

File: tasks/Windows.yml [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+  win_chocolatey:
+    name: chocolatey
+    state: present
+
 - name: Install git
   win_chocolatey:
```
Comment (siong-chin) [inline tasks/Windows.yml:8]: `chocolatey.chocolatey.win_chocolatey` to be consistent?

**Review by siong-chin — APPROVED** [review]

File: tasks/Windows.yml (hunk 1 above)
Comment (sjackson-ccdc) [reply tasks/Windows.yml:2]: There is an issue with the step hanging so I wanted to separate the 2 parts. I agree it should not be necessary. 
Though it does have the added benefit of being more self-documenting this way

File: tasks/Windows.yml (hunk 2 above)
Comment (sjackson-ccdc) [reply tasks/Windows.yml:8]: The linter only fails for Ansible built-in packages. Maybe you're right, I'll do some reading.

**Pull request merged**